src/http/cookie.cr

```diff
@@ -539,5 +539,32 @@ module HTTP
     def to_h : Hash(String, Cookie)
       @cookies.dup
     end
+
+    # Returns a string representation of this cookies list.
+    #
+    # It uses the `Set-Cookie` serialization from `Cookie#to_set_cookie_header` which
+    # represents the full state of the cookie.
+    #
+    # ```
+    # HTTP::Cookies{
+    #   HTTP::Cookie.new("foo", "bar"),
+    #   HTTP::Cookie.new("foo", "bar", domain: "example.com"),
+    # }.to_s # => "HTTP::Cookies{\"foo=bar\", \"foo=bar; domain=example.com\"}"
+    # ```
+    def to_s(io : IO)
+      io << "HTTP::Cookies{"
+      join(io, ", ") { |cookie| cookie.to_set_cookie_header.inspect(io) }
+      io << "}"
+    end
```

Comment:
Usually, you'd see such format in the `inspect` method 🤔

Reply:
This is also used in `#inspect`.
Do you have a suggestion for a more concise format for `#to_s`?

Reply:
For one, I'd move this into the `inspect(io)` impl., since IMO it's more appropriate there.
As for the `to_s` format, ATM I have no better suggestion.